### ISSUE TITLE
Implement unminting funnel in Posthog

### ISSUE THREAD

**Issue by lukasz-zimnoch** [threshold-network/token-dashboard#732]
Depends on: https://github.com/threshold-network/token-dashboard/issues/731

The scope of this issue includes implementing functionality that will send the following unminting-related events to Posthog:
1. Wallet connected (**Connect Wallet** button clicked)
2. Unminting initiated (**Unmint** button clicked)
3. Wallet disconnected (**Disconnect** button clicked)

Additional requirements:
- The ETH address should be captured at point 1 and attached to all subsequent events. Attributing specific events to the given ETH address will allow to re-create specific user journeys
- The event generated at point 1 should contain the information about the user's TBTC balance (1e18 precision)
- The event generated at point 2 should contain the information about the unminted amount (1e18 precision)

**Note: Points 1 and 3 should be already covered in #731 so can be reused here.**

[Relevant Discord thread](https://discord.com/channels/893441201628405760/1022857346789163099/1202945904303284275)
```[tasklist]
### Pull Requests
- [ ] https://github.com/threshold-network/token-dashboard/pull/739
```
